en/concepts/scene/light.md

```diff
@@ -16,6 +16,14 @@ There are two ways to add lights:
 
   ![add light2](light/add-light2.png)
 
+## Additive per-pixel lights
+
+Uber Shader is still the dominant solution on some performance-constrained platforms, but as hardware performance increases and the demand for higher picture quality increases, a fixed number of lights can no longer meet the needs of practical applications, so there is a solution that supports multiple lights -- **Multi-pass drawing**.
```

Comment:
@slackmoehrle Maybe you can refer to this post: https://www.gamedev.net/forums/topic/659145-what-is-a-uber-shader/

Reply:
Thanks, helpful. Do you think we need to give a bit of background or add a link to further documentation in case the user doesn't know what it is? Does it seem like a rather broad, unspecific topic?

Reply:
@slackmoehrle This pr has been delayed for too long, I think we can merge it first, and then submit a new supplement separately.